examples/with-feathersjs/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "with-feathersjs",
+  "description": "",
+  "version": "0.0.0",
+  "homepage": "",
+  "main": "src/",
+  "keywords": [
+    "feathers"
+  ],
+  "license": "MIT",
+  "repository": {},
+  "author": {},
+  "contributors": [],
+  "bugs": {},
+  "engines": {
+    "node": ">= 0.12.0"
+  },
+  "scripts": {
+    "test": "npm run jshint && npm run mocha",
+    "jshint": "jshint src/. test/. --config",
+    ".prestart": "next build",
+    "start": "node src/",
```

Comment:
We recommend to run feathers in a different server. That's the way we need to recommend to create apps.

Use next just for serving the app.
Is it possible to change this into that?